--- x86-mingw32-build.sh.functions.orig	2014-01-23 18:10:26.607946113 +0100
+++ x86-mingw32-build.sh.functions	2014-01-23 18:10:47.951597697 +0100
@@ -126,8 +126,8 @@
 # Some of the following functions can use the `local' built in command,
 # if it is available; make sure we have a fall back, if it isn't ...
 #
-test -z "`exec 2>/dev/null; try(){ local x; }; try || echo no`" || \
-local()
+#test -z "`exec 2>/dev/null; try(){ local x; }; try || echo no`" || \
+#local()
 # ... by making it a function equivalent to the `null' command.
 { :; }
 
